circuit_knitting_toolbox/entanglement_forging/cholesky_decomposition.py

```diff
@@ -92,54 +93,15 @@ def cholesky_decomposition(
             - freeze_shift (float): An energy shift resulting from the decomposition. This shift should be re-applied after
               calculating properties of the decomposed operator (i.e. ground state energy).
     """
-    # Store the ElectronicStructureProblem
-    problem.second_q_ops()
-
-    if problem.grouped_property_transformed is None:
-        raise AttributeError(
-            "There was a problem retrieving the grouped properties from the ElectronicStructureProblem."
-        )
-    if problem.driver is None:
-        raise AttributeError("The ElectronicStructureProblem has no driver.")
-
-    if not isinstance(problem.driver, ElectronicStructureDriver):
-        raise AttributeError(
-            "The ElectronicStructureProblem's driver should be an instance of ElectronicStructureDriver."
-        )
-
-    electronic_basis_transform = problem.grouped_property_transformed.get_property(
-        "ElectronicBasisTransform"
-    )
-    if electronic_basis_transform is None:
-        raise AttributeError(
-            "There was a problem retrieving the ElectronicBasisTransform property from the ElectronicStructureProblem."
-        )
-
-    electronic_energy = problem.grouped_property_transformed.get_property(
-        "ElectronicEnergy"
-    )
-    if electronic_energy is None:
-        raise AttributeError(
-            "There was a problem retrieving the ElectronicEnergy property from the ElectronicStructureProblem."
-        )
-
-    particle_number = problem.grouped_property_transformed.get_property(
-        "ParticleNumber"
-    )
-    if particle_number is None:
-        raise AttributeError(
-            "There was a problem retrieving the ParticleNumber property from the ElectronicStructureProblem."
-        )
-
     # Get data for generating the cholesky decomposition
-    mo_coeff: Matrix = electronic_basis_transform.coeff_alpha
-    hcore: SingleBodyIntegrals = electronic_energy.get_electronic_integral(
-        ElectronicBasis.AO, 1
-    )._matrices[0]
-    eri: TwoBodyIntegrals = electronic_energy.get_electronic_integral(
-        ElectronicBasis.AO, 2
-    )._matrices[0]
-    num_alpha = particle_number.num_alpha
+    driver = PySCFDriver.from_molecule(problem.molecule)
+    driver.run()
+    mo_coeff = driver._calc.mo_coeff
```

Comment:
I'm not really sure how to get the mo coefficients from the problem. I used the driver to get them here